less/index.less

```diff
@@ -4,56 +4,207 @@
-time, mark, audio, video {
-	margin: 0;
-	padding: 0;
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
 }
 /* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
 }
 body {
-	line-height: 1;
+  line-height: 1;
 }
-ol, ul {
-	list-style: none;
+ol,
+ul {
+  list-style: none;
 }
-blockquote, q {
-	quotes: none;
+blockquote,
+q {
+  quotes: none;
 }
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
 }
 table {
-	border-collapse: collapse;
-	border-spacing: 0;
+  border-collapse: collapse;
+  border-spacing: 0;
 }
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
 }
 
 html, body {
-    height: 100%;
+  font-family: 'Thasadith', sans-serif;
+  height: 100%;
+}
+
+.container {
+  width: 100%;
+  max-width: 880px;
+  margin: 0 auto;
+}
+
+.home {
+  margin-top: 20px;
+
+  .top-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 15px;
+    border: 1px solid rgb(53, 28, 22);
+    border-radius: 5px;
+    background: rgb(240, 246, 248);
+
+    h2 {
+      font-size: 2.3rem;
+      color: black;
+      margin-top: 10px;
+    }
+    p {
+      border-top: 1px solid grey;
+      font-size: 1.7rem;
+      color: black;
+      margin-top: 10px;
+      padding: 10px;
+
+    }
+
+    button {
+      transition-duration: 0.3s;
+      color: black;
+      border: 1px solid rgb(53, 28, 22);
+      border-radius: 5px;
+      font-size: 1.5rem;
+      padding: 5px 10px;
+    }
+  }
 }
+  .main-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2%;
+
+    h1 {
+      font-size: 3.6rem;
+      color: rgb(30, 55, 92);
+     }
+    nav a {
+        color: black;
+        padding: 15px;
+        font-size: 1.8rem;
+        text-decoration: none;
+      }
+  }
+
+  .middle-content {
```

Comment:
Good start, make sure you push commits more often so that you can track your progress at more frequent intervals. It will help when you go back to reflect on your work.